packages/react-components/src/popover/src/PopoverTrigger.tsx

```diff
@@ -1,79 +1,89 @@
-import { Children, forwardRef, useCallback } from "react";
+import { Children, ComponentProps, ElementType, ForwardedRef, ReactElement, ReactNode, SyntheticEvent, useCallback } from "react";
 import { Overlay, OverlayArrow, usePopup } from "../../overlay";
 import { PopoverTriggerContext } from "./PopoverTriggerContext";
-import { any, bool, elementType, func, number, oneOf, oneOfType, string } from "prop-types";
-import { augmentElement, mergeProps, resolveChildren } from "../../shared";
+import { augmentElement, forwardRef, mergeProps, resolveChildren } from "../../shared";
 import { isNil } from "lodash";
 
-const propTypes = {
+export interface InnerPopoverTriggerProps {
+    /**
+     * @ignore
+     */
+    id?: string;
```

Comment:
A DomProps interface could always be useful?

https://github.com/adobe/react-spectrum/blob/main/packages/%40react-types/shared/src/dom.d.ts#L47

Reply:
outdated